lib/stdlib/doc/src/peer.xml

```diff
@@ -0,0 +1,319 @@
+          to communicate over distribution.
+        </p>
+      </desc>
+    </func>
+
+    <func>
+      <name name="cast" arity="4" since="OTP 25.0"/>
+      <fsummary>Evaluate a function call on a peer node ignoring the result.</fsummary>
+      <desc>
+        <p>
+          Uses alternative connection to
+          evaluate <c>apply(<anno>Module</anno>, <anno>Function</anno>,
+          <anno>Args</anno>)</c> on the peer node. No response is delivered to the
+          calling process.
+        </p>
+        <p>
+          <c>peer:cast/4</c> fails silently when alternative connection is not
+          configured. Use <seeerl marker="kernel:erpc"><c>erpc</c></seeerl> module
+          to communicate over distribution.
+      </p>
+      </desc>
+    </func>
+
+    <func>
+      <name name="send" arity="3" since="OTP 25.0"/>
+      <fsummary>Sends a message to a process on the peer node.</fsummary>
+      <desc>
+        <p>
+          Uses alternative connection to send <anno>Message</anno> to a process on the
+          the peer node. Silently fails if no alternative connection is configured.
+          Process can be referenced by process ID or registered name.
+        </p>
+      </desc>
+    </func>
+
+    <func>
+      <name name="get_state" arity="1" since="OTP 25.0"/>
+      <fsummary>Returns peer node state.</fsummary>
+      <desc>
+        <p>Returns peer node state. Initial state is <c>booting</c>, node stays in that
+          state until boot script is complete and node progresses to <c>running</c>. If the node
+          stops (gracefully or not), state changes to <c>down</c>.
+        </p>
+      </desc>
+    </func>
+
+    <func>
+      <name name="random_name" arity="0" since="OTP 25.0"/>
+      <fsummary>Creates a sufficiently unique node name.</fsummary>
+      <desc>
+        <p>
+          The same as <seemfa marker="#random_name/1"><c>random_name(peer)</c></seemfa>.
+        </p>
+      </desc>
+    </func>
+
+    <func>
+      <name name="random_name" arity="1" since="OTP 25.0"/>
+      <fsummary>Creates a sufficiently unique node name given a prefix.</fsummary>
+      <desc>
+        <p>
+          Creates a sufficiently unique node name for the current host, combining prefix, current OS
+          process ID and a unique number.
+        </p>
+        <note>
+          <p>
+            Use <c>?CT_PEER(["erl_arg1"])</c> macro provided by Common Test
+            <c>-include_lib("common_test/include/ct.hrl")</c> for convenience. It starts a
+            new peer using Erlang distribution as control channel, supplies calling module
+            code path to the peer, and uses calling function name for name prefix.
+          </p>
+        </note>
+      </desc>
+    </func>
+
+    <func>
+      <name name="start" arity="1" since="OTP 25.0"/>
+      <fsummary>Start a peer node.</fsummary>
+      <desc>
+        <p>Starts a peer node with specified <seetype marker="#start_options"><c>start_options()</c></seetype>.</p>
```

Comment:
I think we will need a short summary of the start options. Which are mandatory? What are the defaults for the others? Will the behavior be similar to the `slave` module or a peer node started with `test_server:start_node()`?

Reply:
All options are optional. I will make a few clarifications regarding the behaviour. In short, when options do not contain alternative connection specification, behaviour is similar to `slave` (because Erlang Distribution connection is providing the heartbeat, as soon as it drops, peer terminates).
When alternative `connection`, behaviour is similar to `test_server:start_node(peer)`.